### PR TITLE
Thatssad/teaching summary

### PR DESCRIPTION
**Pull request by klpittman**

### CODE REVIEW

File: myuw/templates/handlebars/card/summary/section_panel.html [hunk 1]
```diff
@@ -1,17 +1,21 @@
...
             <h4 class="myuw-card-row-heading" style="display:inline-block;
                 margin-bottom:0px;"><div class="c{{color_id}} simplesquare" aria-hidden="true"></div><a href="/teaching/{{ @root.year }},{{ @root.quarter }}#{{ curriculum_abbr }}-{{ course_number }}-{{ section_id }}"
-                class="future-nav-link" future-nav-target="{{@root.year}},{{@root.quarter}},{{curriculum_abbr}}-{{course_number}}-{{section_id}}">{{#unless under_disclosure}}{{curriculum_abbr}} {{/unless}}{{course_number}} {{section_id}}</a></h4>
-            {{#unless under_disclosure}}<span class="meeting-type">{{ meetings.0.type }}</span>{{/unless}}
+                class="future-nav-link" future-nav-target="{{@root.year}},{{@root.quarter}},{{curriculum_abbr}}-{{course_number}}-{{section_id}}">{{curriculum_abbr}} {{course_number}} {{section_id}}</a></h4>
+            <span class="meeting-type">{{ meetings.0.type }}</span>
         </div>
 
         <div class="col-sm-9">
+        {{/unless}}
```
Comment (fanglinfang) [inline myuw/templates/handlebars/card/summary/section_panel.html]: should this end of {{/unless}} be on line 11?

File: myuw/dao/instructor_schedule.py [hunk 2]
```diff
@@ -191,4 +191,4 @@ def check_section_instructor(section, person=None):
 def exceeded_max_display_sections(total_sections):
     return total_sections > getattr(settings,
                                     "MYUW_MAX_INSTRUCTOR_SECTIONS",
-                                    10)
+                                    3)
```
Comment (fanglinfang) [inline myuw/dao/instructor_schedule.py]: please change this back to 10

File: myuw/templates/handlebars/card/summary/section_panel.html (hunk 1 above)
Comment (klpittman) [reply myuw/templates/handlebars/card/summary/section_panel.html]: Yes, essentially I'm putting the top level classes into our grid system, while trying to take classes that are in the disclosures out of a grid system and only into a table.

**Comment by fanglinfang** [comment]
does

**Pull request merged**